stdlib/contrib/bonitoo-io/tickscript/README.md

```diff
@@ -0,0 +1,385 @@
+# Tickscript package
+
+The `tickscript` package can be used to convert TICKscripts to InfluxDB 2.x tasks.
+
+Most TICKscript functions have the same or similar counterparts in Flux.
+This package only provides a set of convenience functions for easier conversion and creation of custom `checks` executed as `tasks` and triggering `alerts` in InfluxDB 2.x.
+
+To learn more about monitoring and alerting in InfluxDB 2.x and Flux, please see
+ - [Monitor data and send alerts](https://docs.influxdata.com/influxdb/v2.0/monitor-alert/)
+ - [Flux InfluxDB Monitor package](https://docs.influxdata.com/influxdb/v2.0/reference/flux/stdlib/monitor/)
+ - [Process Data with InfluxDB tasks](https://docs.influxdata.com/influxdb/v2.0/process-data/)
+
+## Available functions
+
+- `select`
+- `selectWindow`
+- `groupBy`
+- `compute`
+- `join`
+- `alert`
+- `deadman`
+- `defineCheck`
+
+## Conversion guidelines
+
+* Variable conversions  
+  `var realm = 'qa'` becomes `realm = "qa"` in Flux  
+  `var warnLevel = lambda: "device_count" > 2000` is a function `warnLevel = (r) => r["device_count"] > 2000` in Flux
+
+* Both `batch` and `stream` translates to `from(bucket: ...)` in Flux.
+
+* Every `batch`, `stream` and `deadman` must be separate task.
+
+* `every(duration)` and `offset(duration)` property maps to `every` and `offset` fields in task's option record.
+  For better control or aligned scheduling, use `cron` option instead.
+
+* `period(duration)` property maps to `range(start: -duration)` in Flux pipeline.
+
+* `deadman()`'s `interval` maps to task's `every` scheduling value.
+
+* `groupBy(columns)` maps to `group(columns)`.  
+  Columns must include internal `_measurement` column.
+  For convenience, function `groupBy` is provided in this package.  
+  Flux results are grouped by all tags by default. To ungroup, call `group()` without argument.
+
+```js
+    query('''
+        SELECT mean("counter") AS "total_mean" WHERE ...
+        ''')
+        .groupBy('host')
+        .period(1m)
+```
+can be rewritten to Flux as
+```js
+    from(bucket: ...)
```

Comment:
nit but can you add the `import` statement here so the Flux code would be complete?